server/utils/README.md

```diff
@@ -1,7 +1,7 @@
-### **The utils directory will serve as a location for mutltiple utilty tools.  get_request_data_csv.py can be used to make an API call to fetch 311 request data from the 311 data server for a given start date and end date.** 
+### The utils directory will serve as a location for mutltiple utilty tools.  get_request_data_csv.py can be used to make an API call to fetch 311 request data from the 311 data server for a given start date and end date.
```

Comment:
Let's just remove all of the "###"s here. We don't want anything to be bolded.

Reply:
Sounds good! Thank you Nich!